### PR TITLE
NCL_native_1.py

### LINKED ISSUE #303: native_1
Description: An example of a Stereographic native grid

NCL script: https://www.ncl.ucar.edu/Applications/Scripts/native_1.ncl
NCL plot: https://www.ncl.ucar.edu/Applications/Images/native_1_lg.png

Category: MapProjections

### PR DESCRIPTION
**Pull request by hCraker**
Closes #303 
NCL Plot: https://www.ncl.ucar.edu/Applications/Images/native_1_lg.png

Current state of plot
image

### CODE REVIEW

**Comment by hCraker** [comment]
image

**Review by anissa111 — APPROVED** [review]
I noted a typo below, but other than that, it looks good to go! I like what you did with the lat lon lines!

File: Plots/MapProjections/NCL_native_1.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+# Specify the indices of the desired colors
+index = [0, 200, 180, 160, 140, 120, 100, 80, 60, 40, 20, 0]
+color_list = [cmap[i].colors for i in index]
+
+# make the starting color and end color white
+color_list[0] = [1, 1, 1]  # [red, green, blue] values range from 0 to 1
+color_list[-1] = [1, 1, 1]
+
+
+# Plot contour data, use the transform keyword to speficy that the data is
```
Comment (anissa111) [inline Plots/MapProjections/NCL_native_1.py:90]:
```suggestion
# Plot contour data, use the transform keyword to specify that the data is
```

File: Plots/MapProjections/NCL_native_1.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+# Manipulate latitude and longitude gridline numbers and spacing
+gl.xlocator = mticker.FixedLocator(np.arange(4, 18, 2))
+gl.ylocator = mticker.FixedLocator(np.arange(43, 50))
+gl.xlabel_style = {"rotation": 0, "size": 14}
+gl.ylabel_style = {"rotation": 0, "size": 14}
+
+# Create colormap by choosing colors from existing colormap
+# The brightness of the colors in cmocean_speed increase linearly. This
+# makes the colormap easier to interpret for those with vision imparements
```
Comment (anissa111) [inline Plots/MapProjections/NCL_native_1.py:78]:
```suggestion
# makes the colormap easier to interpret for those with vision impairments
```

**Review by michaelavs — APPROVED** [review]
Looks good to me after making the changes requested by Anissa!

**Comment by hCraker** [comment]
Hi all. It looks like this was merged before I had a chance to fix those typos. Should I open a new PR for that or can I make those changes here?

**Comment by michaelavs** [comment]
> 

I think you can click 'revert changes' and it should reopen the PR (or at least create a new one). If that doesn't work, I would just make a new PR with the changes and we can get it reviewed/merged again pretty quickly

**Comment by michaelavs** [comment]
> Hi all. It looks like this was merged before I had a chance to fix those typos. Should I open a new PR for that or can I make those changes here?

I think you can click 'revert changes' and it should reopen the PR (or at least create a new one). If that doesn't work, I would just make a new PR with the changes and we can get it reviewed/merged again pretty quickly

**Comment by erogluorhan** [comment]
> Hi all. It looks like this was merged before I had a chance to fix those typos. Should I open a new PR for that or can I make those changes here?

Hi @hCraker you don't need to do anything since I merged it and then corrected those typos in the next merge of another PR. Check the code in `main` branch and you will see they are corrected. Let me know if I missed anything though

**Comment by hCraker** [comment]
> Hi @hCraker you don't need to do anything since I merged it and then corrected those typos in the next merge of another PR. Check the code in `main` branch and you will see they are corrected. Let me know if I missed anything though

@erogluorhan I see that you fixed the ones that @anissa111 pointed out. There is also a typo in line 12 for the word accommodate.

**Comment by erogluorhan** [comment]
> > Hi @hCraker you don't need to do anything since I merged it and then corrected those typos in the next merge of another PR. Check the code in `main` branch and you will see they are corrected. Let me know if I missed anything though
> 
> @erogluorhan I see that you fixed the ones that @anissa111 pointed out. There is also a typo in line 12 for the word accommodate.

Yeah, I only corrected those Anissa found. Please create a simple new PR that handles typo corrections.

**Pull request merged**